### ISSUE TITLE
Clicking a tab in reference changes focus of all tabs

### ISSUE THREAD

**Issue by brian-avery** [istio/istio.io#7770]
Clicking a tab in the reference content changes the focus of all tabs. 

For example, open https://istio.io/latest/docs/reference/config/networking/service-entry/

Click on `v1beta1` in one of the tab sets. Observe that all tabs now navigate to the other option. I haven't dug into this much, but presumably this could be fixed by generating a unique identifier for each tab.

**Comment by ericvn** [comment]
Note that I sometimes found this useful when working through some examples with had tabs for various paths. Choosing a particular path in one tabset automatically choose the same path in another tabset.

**Comment by ericvn** [comment]
Although, I am lacking on finding an example of my last comment in the 1.6 pages. I suspect they may have been in the 1.5 or 1.4 multicluster pages.

**Comment by brian-avery** [comment]
Yeah, on some pages it might make sense. Seemed like an odd behavior to me, but, say, on the ingress page it might be useful.
